packages/twenty-server/src/integrations/tracing/useTracing.ts

```diff
@@ -0,0 +1,31 @@
+import * as Sentry from '@sentry/node';
+import { handleStreamOrSingleExecutionResult, Plugin } from '@envelop/core';
+import { OperationDefinitionNode, Kind } from 'graphql';
+
+export const useTracing = (): Plugin => {
+  return {
+    onExecute({ args }) {
+      console.log('onExecute');
+      console.log(args);
+      const transactionName = args.operationName || 'Anonymous Operation';
+      const rootOperation = args.document.definitions.find(
+        (o) => o.kind === Kind.OPERATION_DEFINITION,
+      ) as OperationDefinitionNode;
+      const operationType = rootOperation.operation;
+
+      if (Sentry.isInitialized()) {
```

Comment:
@FelixMalfait Maybe we should do something like `useExceptionHandler` and pass a service with custom drivers for tracing. So we can support multiple driver for tracing

Reply:
I've considered it and started with this initially. There is a common driver for tracing which is called OpenTelemetry, and Sentry is implementing it. But there are several reasons why I deleted my implementation and went back to this. 
The 2 majors were that OpenTelemetry doesn't support profiling yet. And there is no case for enabling people to setup profiling on their own instance - this is really something specific to us. Also the instrumentation lib for Sentry pg worked best for me than the otel implementation.